reana-ui/src/pages/signin/Signin.js

```diff
@@ -77,7 +77,13 @@ export default function Signin() {
           />
         )}
       </Segment>
-      {config.hideSignup && (
+      {config.hideSignup && config.cernSSO && (
+        <p>
+          Note that you need to hold an official CERN account in order to use
+          <Link to="/"> reana.cern.ch</Link> service.
```

Comment:
What do you think about using `window.location.hostname` instead of `reana.cern.ch`?

Reply:
I agree that it's not too nice to hardcode it like this, my thinking was that for `cernSSO` case this value most likely will never change. Ideally we would have a config for this coming from the api, for now I've switched it to `window.location.hostname` as you suggested, thanks